ProcessLib/HT/CreateHTProcess.cpp

```diff
@@ -115,6 +115,25 @@ std::unique_ptr<Process> createHTProcess(
     DBUG("Use \'%s\' as specific_heat_capacity_fluid parameter.",
          specific_heat_capacity_fluid.name.c_str());
 
+
+    // Parameter for the thermal conductivity of the solid (only one scalar per
+    // element, i.e., the isotropic case is handled at the moment)
+    auto& thermal_dispersivity_longitudinal = findParameter<double>(
```

Comment:
If thermal_conductivity is input as matrix,  thermal dispersivity data can be input as  thermal_conductivity directly. Or you keep the  thermal dispersivity input but add the values to thermal_conductivity matrix data directly after parsing the input.